conf/db/migration/metrics_portal_ddl/postgresql/V3__create_alerts_and_expressions_table.sql

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2015 Groupon
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CREATE TYPE context AS ENUM ('HOST', 'CLUSTER');
+CREATE TYPE operator AS ENUM ('EQUAL_TO', 'NOT_EQUAL_TO', 'LESS_THAN', 'LESS_THAN_OR_EQUAL_TO', 'GREATER_THAN', 'GREATER_THAN_OR_EQUAL_TO');
+CREATE TABLE portal.alerts (
+    id UUID PRIMARY KEY,
+    version INTEGER NOT NULL DEFAULT 1,
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP NOT NULL DEFAULT now(),
+    name VARCHAR(255) NOT NULL,
+    cluster VARCHAR(255) NOT NULL,
+    service VARCHAR(255) NOT NULL,
+    context context NOT NULL,
+    metric VARCHAR(255) NOT NULL,
+    statistic VARCHAR(255) NOT NULL,
+    period VARCHAR(30) NOT NULL,
```

Comment:
Here are a few points for example that might get you started:

 - con: it is harder and slower to query with any relative operation on the period using iso8601 since you need to parse it assuming such a facility exists in postgresql
 - pro: it's in a general format understood by the application
 - con: it's more expensive to store as iso-8601
 - pro: the format is less ambiguous (than say seconds) since it embeds the units